package.json

```diff
@@ -27,7 +27,14 @@
     "require": [
       "babel-register",
       "babel-polyfill"
-    ]
+    ],
+    "babel": {
+      "presets": [
+        "es2015",
+        "stage-3",
+        "react"
+      ]
+    }
```

Comment:
No worries man, this is open source and I am not your boss, there is no deadline :D 
Thanks again for contributing